example/chunks/node/Dockerfile

```diff
@@ -1,5 +1,13 @@
+
 ARG base
 FROM ${base}
 
-RUN apk add --no-cache nodejs
-ENV LANG=en_US.UTF-8
\ No newline at end of file
+ARG NODE_VERSION
+
+RUN curl -fsSL https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | PROFILE=/dev/null bash
```

Comment:
I know it is an example but how are we intending to handle versioning/upgrading nvm going forward? An nvm chunk?

Reply:
Good point. I've introduced an `NVM_VERSION` arg that we could make variant if we wanted to